src/SonarQube.TeamBuild.PostProcessor/CoverageReportProcessor.cs

```diff
@@ -60,7 +60,8 @@ public bool ProcessCoverageReports()
         {
             Debug.Assert(initialisedSuccesfully, "Initialization failed, cannot process coverage reports");
 
-            return processor.ProcessCoverageReports();
+            // If we return false then processing will stop so if in doubt return true
+            return processor?.ProcessCoverageReports() ?? true;
```

Comment:
I didn't want to make any changes to the product code since we've done all of the testing and are about to release.
However, this code path would have crashed if it had every been hit so I've added a defensive check so that processing will continue.